android/envoy/src/main/kotlin/org/greatfire/envoy/CronetNetworking.kt

```diff
@@ -22,10 +22,9 @@ object CronetNetworking {
 
     fun buildEngine(
         context: Context,
-        cacheFolder: String? = null,
-        envoyUrl: String? = null,
-        proxyUrl: String? = null,
-        resolverRules: String? = null,
+        cacheFolder: String? = "",
+        proxyUrl: String? = "",
+        resolverRules: String? = "",
```

Comment:
if these are being initialized to a non-null value, can you make them non-nullable?

Reply:
Good call, I don't remember if Cronet was barfing on the nulls or if it was something I tried that didn't help, but it's probably just as well to pass empty strings (I'm kinda looking at the patches, I'll verify how they handle null vs empty string)